packages/website/pages/pricing.js

```diff
@@ -1,34 +1,405 @@
+          <div className="column-content">
+            <div className="block image-block">
+              <Img src="/images/logos/nft-storage.svg" alt="NFT Storage" />
+              <div className="image-label"></div>
+            </div>
+            <div className="block image-block">
+              <Img src="/images/logos/protocol-labs.svg" alt="Protocol Labs" />
+              <div className="image-label"></div>
+            </div>
+            <div className="block image-block">
+              <Img src="/images/logos/filecoin.svg" alt="Filecoin" />
+              <div className="image-label"></div>
+            </div>
+            <div id="intro_1-triangle_left">
+              <Img alt="" src={ImageTriangle1} />
+            </div>
+            <div id="intro_1-triangle_right">
+              <Img alt="" src={ImageTriangle} />
+            </div>
+            <div id="intro_1-cross_right">
+              <Img alt="" src={ImageCross} />
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  </>
+);
 
-export default function Home() {
-  const sections = TiersPageData.page_content;
-  const animations = TiersPageData.floater_animations;
-
-  useEffect(() => {
-    let pageFloaters = {};
-    if (animations) {
-      initFloaterAnimations(animations).then(result => {
-        pageFloaters = result;
-      });
-    }
-    return () => {
-      if (pageFloaters.hasOwnProperty('destroy')) {
-        pageFloaters.destroy();
-      }
-    };
-  }, [animations]);
+const Card = props => {
+  const { title, price, isBestValue, children, storageAllocation, storageOverageRate, callToAction, callToActionUrl } =
+    props;
+  return (
+    <div className="pricing-card">
+      {!!isBestValue && <div className="best-value-adornment">Best Value!</div>}
+      <div className={`pricing-card-body${isBestValue ? ' adorned' : ''}`}>
+        <div className="pricing-title-row">
+          <div className="pricing-title">{title}</div>
+          <div className="pricing-price">
+            {price}
+            <div className="pricing-price-term">/month</div>
+          </div>
+        </div>
+        <ul className="plan-details">{children}</ul>
+        <div className="plan-summary">
+          <div className="plan-storage-allocation">{storageAllocation}</div>
+          <div className="plan-overage-rate">{storageOverageRate}</div>
+          <a href={callToActionUrl}>
+            <div className="plan-call-to-action button dark Button">{callToAction}</div>
+          </a>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const PricingTiers = () => (
+  <>
+    <section id="section_plan_cards" className="sectional">
+      <div className="grid-middle">
+        <div className="col-12_sm-8_mi-10_ti-12 column-1" data-push-left="off-0_sm-2_mi-1_ti-0">
+          <div className="column-content">
+            <Card
+              title="Free"
+              price="$0"
+              storageAllocation="5GiB storage"
```

Comment:
wdyt of getting this number '5' from the [plansContext](https://github.com/web3-storage/web3.storage/blob/main/packages/website/components/contexts/plansContext.js#L34) that @cmunns set up? That way the values don't get out of sync?

If good idea but it would take too long, maybe just make another issue to track followup? I think we'll iterate a bit on where the data comes from, maybe even eventually get the plan info not from the website codebase but at runtime via an http api like /storage/prices that returns json describing each price tier

Reply:
I didn't know plansContext existed, but that looks perfect